tests/logictest/main.py

```diff
@@ -6,36 +6,50 @@
 from http_runner import TestHttp
 from clickhouse_runner import TestClickhouse
 
+from argparse import ArgumentParser
 from config import mysql_config, http_config, clickhouse_config
 
-import fire
 
-
-def run(pattern=".*"):
+def run(args):
     """
     Run tests
     """
-    disable_mysql_test = os.getenv("DISABLE_MYSQL_LOGIC_TEST")
-    if disable_mysql_test is None:
-        mysql_test = TestMySQL("mysql", pattern)
+    if not args.disable_mysql_test:
+        mysql_test = TestMySQL("mysql", args.pattern)
         mysql_test.set_driver(mysql_config)
         mysql_test.set_label("mysql")
         mysql_test.run_sql_suite()
 
-    disable_http_test = os.getenv("DISABLE_HTTP_LOGIC_TEST")
```

Comment:
Arg from env need by cloud platform.keep it or add defaut value from env.

Reply:
no need, it's default to None as before.